assets/js/validasi.js

```diff
@@ -94,6 +94,9 @@ $(document).ready(function() {
 	});
 
 	$("#validasi").validate({
+		rules: {
+			nama: {nomor_surat_keputusan: true}
+		},
```

Comment:
Sebaiknya tidak ditambahkan ke sini, karena #validasi digunakan di banyak form. Jadi akan mempengaruhi banyak form. Alternatifnya adalah menambahkan validasi class `nomor_surat_keputusan` langsung di kolom isian ybs di masing2 form.

Reply:
@eddieridwan saya baru tahu ternyata rule validasi bisa dipasang melalui class. ini tutorialnya dari mana ya pak? saya kemarin baca2 di dokumentasi jquery validation kayaknya ngga ada contoh seperti ini.

di validasi.js juga ada baris kode untuk menambahkan rule validasi ke komponen html, tapi kayaknya ngga jalan. seperti yang ini:

	$('.nama').each(function() {
		$(this).rules("add",
			{
				nama: true,
			});
	});

Reply:
Ya, memang fitur itu ditemukan melalui posting oleh komunitas pengguna plugin validate, bukan dokumentasi resmi plugin itu. Tapi sepertinya implementasi plugin validate juga menggunakan script seperti 
```
	$('.nama').each(function() {
		$(this).rules("add",
			{
				nama: true,
			});
	});
```
yang seharusnya jalan.